security.tf

```diff
@@ -0,0 +1,29 @@
+# Security Groups
+resource "aws_security_group" "this" {
```

Comment:
Let's make sure this is exposed as an output

Reply:
We already have this output.